apps/sports-helsinki/src/domain/venue/venueInfo/VenueInfo.tsx

```diff
@@ -17,11 +17,10 @@ import styles from './venueInfo.module.scss';
 
 const OpeningHoursInfo = ({ venue: { connections } }: { venue: Venue }) => {
   const { t } = useVenueTranslation();
+  const sectionTypes = ['OPENING_HOURS', 'OPENING_HOUR_OBJECT'];
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const connectionOpeningHoursSectionsContents: any[] = connections
-    ?.filter(
-      (item) => item?.sectionType === 'OPENING_HOURS' || 'OPENING_HOUR_OBJECT'
-    )
+    ?.filter((item) => sectionTypes.includes(item?.sectionType || ''))
```

Comment:
the `sectionTypes` is static here, so it can never include `''` or `null` or `undefined`. So, the `|| ''` is needless

Reply:
if it's there only to fool TS, it's' fine.